tests/unit/test_pybamm_data.py

```diff
@@ -0,0 +1,36 @@
+import pybamm
+import pytest
+import random
+import requests
```

Comment:
I think it is a required dependency of pooch

Reply:
https://www.fatiando.org/pooch/latest/install.html

Reply:
Makes sense. I am not sure if we should include it as well – explicit is better than implicit so I would say we should, but happy to hear from others as well

Reply:
Here, I would say we should use something from the standard library, like

```python
import socket

host = socket.gethostbyname("www.github.com")
conn = socket.create_connection((host, 80), 2)
conn.close()
```

inside a try-except block rather than importing a transitive dependency like `requests` that we do not explicitly mark as ours. This should be reasonable and diverts the question altogether, plus it helps us in case `pooch` ever decides to drop `requests` and aims to download files using plain-old `urllib` (not that it seems it will, though).

Reply:
Sure that would work, I'll do that then!